### PR TITLE
[Nº3] Initial screen

### PR DESCRIPTION
**Pull request by micaelacole**
## Summary ##

**Trello card**: https://trello.com/c/FHnCIlK2/23-n3-pantalla-de-inicio

Initial screen of the app with a "Log in with Google" button.

### iPhone 8

iphone8

### iPhone 11 Pro Max

11promax

## Screencast ##

This is how the login button works in iPhone 11 Pro Max:

11promax

It was tested on the rest of the devices aswell.

### CODE REVIEW

File: WBooks/LandingScreen/LandingScreenView.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
+//
+//  LandingScreenView.swift
+//  WBooks
+//
+//  Created by Megan Cole on 08/01/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import SwiftUI
```
Comment (matiasspinelli4) [inline WBooks/LandingScreen/LandingScreenView.swift]: Why import SwiftUI? Is this a default from Xcode 11? I guess this application runs on iOS 10, so none of SwiftUI's new things will be implemented = (

**Comment by matiasspinelli4** [comment]
Try change the constraints of the LandingScreen bottom or top from Superview to Safe Area for proper implementation on iPhone X devices

**Comment by matiasspinelli4** [comment]
Nice Job! 💪🏻

**Comment by gmazzei** [comment]
To fix the gaps that appear on iPhone X, check that the constrains reach the limits of the screen, as Mati mentioned. If you already did that, change `contentMode` to "Aspect Fill". That will probably do the trick 😉

**Pull request merged**